it/common/src/main/java/org/apache/beam/it/common/conditions/package-info.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/** Package that contains reusable conditions. */
+package org.apache.beam.it.common.conditions;
```

Comment:
Don't make subpackages of `common`. Common is already "misc" so you can just put `conditions` at the top level.